### PR TITLE
graylogPlugins: Add and update plugins

### PR DESCRIPTION
**Pull request by fadenb**
###### Motivation for this change
Add and update plugins

###### Things done

- [x] Tested using sandboxing
  ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS,
    or option `build-use-sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file)
    on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] Linux
- [x] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

File: pkgs/tools/misc/graylog/plugins.nix [hunk 1]
```diff
@@ -35,6 +35,19 @@ in {
       description = "SSO support for Graylog through trusted HTTP headers set by load balancers or authentication proxies";
     };
   };
+  internal-logs = glPlugin rec {
+    name = "graylog-internal-logs-${version}";
+    pluginName = "graylog-plugin-internal-logs";
+    version = "1.0.0";
+    src = fetchurl {
```
Comment (FRidh) [inline pkgs/tools/misc/graylog/plugins.nix]: fetchFromGitHub

File: pkgs/tools/misc/graylog/plugins.nix [hunk 2]
```diff
@@ -61,6 +74,19 @@ in {
       description = "Jabber Alarmcallback Plugin for Graylog";
     };
   };
+  mongodb-profiler = glPlugin rec {
+    name = "graylog-mongodb-profiler-${version}";
+    pluginName = "graylog-plugin-mongodb-profiler";
+    version = "2.0.1";
+    src = fetchurl {
```
Comment (FRidh) [inline pkgs/tools/misc/graylog/plugins.nix]: fetchFromGitHub

File: pkgs/tools/misc/graylog/plugins.nix (hunk 1 above)
Comment (fadenb) [inline pkgs/tools/misc/graylog/plugins.nix]: @FRidh I do not know how I could use fetchFromGitHub to fetch files released on GitHub. Is it possible to fetch the released files (not the source)?

Comment (FRidh) [inline pkgs/tools/misc/graylog/plugins.nix]: I thought you did refer to the source. You can ignore my comment.

**Pull request merged**